flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/failover/flip1/RestartPipelinedRegionFailoverStrategy.java

```diff
@@ -98,7 +98,7 @@ public RestartPipelinedRegionFailoverStrategy(
 	// ------------------------------------------------------------------------
 
 	private void buildFailoverRegions() {
-		final Set<? extends Set<? extends SchedulingExecutionVertex>> distinctRegions =
+		final Set<? extends Set<SchedulingExecutionVertex>> distinctRegions =
```

Comment:
It can be further simplified to `Set<Set<SchedulingExecutionVertex>>`.
The same applies to similar changes in RegionPartitionReleaseStrategy and PipelinedRegionComputeUtil.

Reply:
The for-loop below can be simplified as well.
`for (Set<? extends SchedulingExecutionVertex> regionVertices : distinctRegions) {`

Reply:
Fixed. The remaining wildcards look legitimate to me

<img width="1296" alt="image" src="https://user-images.githubusercontent.com/1681921/79549620-864f9180-8097-11ea-97d8-1f3b7a45307e.png">